### ISSUE TITLE
No keyboard shortcuts available

### ISSUE THREAD

**Issue by Krish1908** [KRTirtho/spotube#2122]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I mostly use my laptop for listening to music. Spotify had the keyboard shortcuts to skip the song, go to previous song, increase or decrease volume, seek back and forth. That shortcut is lacking here in spotube.

### Suggested solution

I suggest you to implement the same shortcut keys here on spotube.
![Spotify Keyboard Shortcuts](https://github.com/user-attachments/assets/8fb915a9-1a6e-4598-ad49-c7c001d0d636)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by eeriemyxi** [comment]
Yes please. I have no idea how others managed to use Spotube on desktops without shortcuts thusfar but I recently switched to using Spotube on my desktop and this is incredibly painful to experience. I also noticed that it doesn't register itself as a media player through the [MPRIS interface](https://specifications.freedesktop.org/mpris-spec/latest/), so the shortcuts that I use to play-pause stuff universally on my Desktop doesn't work. This is also important for accessibility purposes.
